upb/msg.c

```diff
@@ -0,0 +1,1053 @@
+
+#include "upb/msg.h"
+
+static bool is_power_of_two(size_t val) {
```

Comment:
I prefer to keep them local until/unless they are needed in more than one place.